docs/UserGuide.md

```diff
@@ -308,158 +308,158 @@ the constraints of each parameter when used in a command.
 
 #### Adding a patient: `add`
 
-Adds a patient and their relevant details to MediBase3.
+Adds a patient and his/her relevant details to MediBase3.
 
-Format: `add n/NAME i/NRIC g/GENDER d/DOB p/PHONE_NUMBER e/EMAIL a/ADDRESS`
+**Format**: `add n/NAME i/NRIC g/GENDER d/DOB p/PHONE_NUMBER e/EMAIL a/ADDRESS`
 
 {: .alert .alert-info}
 > :information_source: **Note:**
 > 
 > * All fields are compulsory and must be non-empty.
-> * A patient will not be added if the NRIC given is already associated with another patient in MediBase3. An error message will be displayed in this case.
+> * If the `NRIC` provided already exists in MediBase3, an error message will display, and the patient will not be added.
 > * The new patient will be added to the end of the Patient List Panel.
 > * Refer to the [Parameter Details](#parameter-details) section for more information on the purpose and constraints of each parameter.
 
-Examples:
+**Examples**:
 * `add n/John Doe i/S1234567A g/M d/2002-12-12 p/98765432 e/johnd@example.com a/Orchard Road, Block 124, #02-01`
 * `add n/Betsy Crowe i/s1234567b g/F e/betsycrowe@example.com a/Bukit Merah, Block 123, #01-01 p/1234567 d/2002-11-10`
 
 {: .alert .alert-success}
 > :bulb: **Tip:**
 > 
 > * Remember that `NRIC` and `GENDER` are case-insensitive.
-> E.g. `i/s1234567a` and `i/S1234567A` are both equivalent.
+>   -  e.g. `i/s1234567a` and `i/S1234567A` are both equivalent.
 > * Made a mistake or a typo? You can use the [`edit` command](#editing-a-patient--edit) to update the patient's details.
 
 [Back to Table of Contents](#table-of-contents)
 
 #### Deleting a patient : `delete`
 
-Deletes the specified patient and their details from MediBase3.
+Removes a patient and his/her details from MediBase3.
 
-Format: `delete NRIC`
+**Format**: `delete NRIC`
 
 {: .alert .alert-info}
 > :information_source: **Note:**
 > 
-> * Deletes the patient with the specified `NRIC` in MediBase3.
-> * The `NRIC` provided must be the **full NRIC** of the patient to be deleted. e.g. `S1234567A` and not `S123`.
-> * You can delete a patient even if they are not being currently displayed in the Patient List Panel.
-> * `NRIC` must adhere to the constraints mentioned in the [Parameter Details](#Parameter-Details) section.
+> * Deletes the patient with the specified `NRIC` from MediBase3.
+> * The `NRIC` provided must be complete, 
+>   - e.g. `S1234567A` (not partial, like `S123`).
+> * You can delete a patient even if they’re not currently displayed in the Patient List Panel.
```

Comment:
Should the "but doing so will refresh the..." part for consistency?

Reply:
I felt that part was repetitive so I removed. But maybe I didn't change everything hmm

Reply:
I feel that its an important information since the user might not know why after typing a command with the list being filtered, it changes back to show all the patients

Reply:
At most, you can mention that all commands (except the ones under Finding Patients) will list all the patients after it is executed successfully

Reply:
Hmmm. what are your thoughts on a 'global' note at the start of the command section, like

{: .alert .alert-info}
> :information_source: **General Behavior:**
>
> All commands (except those under **Finding Patients**) will refresh the Patient List Panel to show all patients upon successful execution. This includes commands for managing patients, appointments, medical conditions, allergies, and priority.
>
> **Note:** Commands under **Finding Patients** only filter the displayed patients without refreshing the full Patient List Panel.


@yadobler @yong-zhuo

Reply:
I think the Note with regards to finding patient might not be necessary since i alr put a note at the start of Finding Patient section to mention abt that